Run-AlPipeline latest preview not working (bcinsider)

This weekend testing our apps for BC25 stopped working.
I we use the latest preview version, and I noticed this was updated saturday.
The error 'Cannot bind argument to parameter 'Path' because it is an empty string' is something common,
but I do not think that this time it is on my side, because nothing has changed in the scripts we use for
spinning up bc container helper.


**Scripts used to create container and cause the issue**
```
        Run-AlPipeline `
            -azureDevOps `
            -doNotRunTests `
            -accept_insiderEula `
            -credential $credential `
            -artifact $artifactUrl `
            -pipelineName $PipeLineName `
            -licenseFile $LicFileName `
            -baseFolder $BaseFolder `
            -appFolders $AppFolders `
            -installApps $installApps `
            -PreviousApps $PreviousApps `
            -keepContainer `
            -imageName $ImageName  
```

**Full output of scripts**
```
2024-06-02T22:32:58.1397089Z  |  __ \                             | |
2024-06-02T22:32:58.1397253Z  | |__) |_ _ _ __ __ _ _ __ ___   ___| |_ ___ _ __ ___
2024-06-02T22:32:58.1397773Z  |  ___/ _` | '__/ _` | '_ ` _ \ / _ \ __/ _ \ '__/ __|
2024-06-02T22:32:58.1397943Z  | |  | (_| | | | (_| | | | | | |  __/ |_  __/ |  \__ \
2024-06-02T22:32:58.1399983Z  |_|   \__,_|_|  \__,_|_| |_| |_|\___|\__\___|_|  |___/
2024-06-02T22:32:58.1407419Z 
2024-06-02T22:32:58.1428449Z Pipeline name                   projectx
2024-06-02T22:32:58.1450039Z Container name                  projectx-bld
2024-06-02T22:32:58.1466581Z Image name                      
2024-06-02T22:32:58.1486159Z ArtifactUrl                     https://bcinsider-fvh2ekdjecfjd6gk.b02.azurefd.net/sandbox/25.0.20320.0/w1
2024-06-02T22:32:58.1503299Z BcAuthContext                   Not Specified
2024-06-02T22:32:58.1519563Z Environment                     
2024-06-02T22:32:58.1541670Z ReUseContainer                  False
2024-06-02T22:32:58.1572435Z KeepContainer                   True
2024-06-02T22:32:58.1592339Z useCompilerFolder               False
2024-06-02T22:32:58.1609516Z artifactCachePath               
2024-06-02T22:32:58.1627040Z useDevEndpoint                  False
2024-06-02T22:32:58.1647021Z Auth                            UserPassword
2024-06-02T22:32:58.1664436Z CompanyName                     
2024-06-02T22:32:58.1686088Z MemoryLimit                     8G
2024-06-02T22:32:58.1708289Z FailOn                          none
2024-06-02T22:32:58.1727392Z TreatTestFailuresAsWarnings     False
2024-06-02T22:32:58.1746824Z Enable Task Scheduler           False
2024-06-02T22:32:58.1765854Z Assign Premium Plan             False
2024-06-02T22:32:58.1786744Z Install Test Runner             False
2024-06-02T22:32:58.1805347Z Install Test Framework          False
2024-06-02T22:32:58.1823678Z Install Test Libraries          False
2024-06-02T22:32:58.1843041Z Install Perf. Toolkit           False
2024-06-02T22:32:58.1864847Z InstallOnlyReferencedApps       False
2024-06-02T22:32:58.1886232Z generateDependencyArtifact      False
2024-06-02T22:32:58.1904874Z CopySymbolsFromContainer        False
2024-06-02T22:32:58.1923428Z enableCodeCop                   False
2024-06-02T22:32:58.1941762Z enableAppSourceCop              False
2024-06-02T22:32:58.1959896Z enableUICop                     False
2024-06-02T22:32:58.1997636Z enablePerTenantExtensionCop     False
2024-06-02T22:32:58.2018706Z enableCodeAnalyzersOnTestApps   False
2024-06-02T22:32:58.2042634Z doNotPerformUpgrade             False
2024-06-02T22:32:58.2062500Z doNotPublishApps                False
2024-06-02T22:32:58.2083146Z uninstallRemovedApps            False
2024-06-02T22:32:58.2100198Z escapeFromCops                  False
2024-06-02T22:32:58.2120246Z doNotBuildTests                 False
2024-06-02T22:32:58.2141329Z doNotRunTests                   True
2024-06-02T22:32:58.2163164Z doNotRunBcptTests               False
2024-06-02T22:32:58.2183244Z useDefaultAppSourceRuleSet      False
2024-06-02T22:32:58.2198343Z rulesetFile                     
2024-06-02T22:32:58.2220552Z generateErrorLog                False
2024-06-02T22:32:58.2240533Z enableExternalRulesets          False
2024-06-02T22:32:58.2257185Z azureDevOps                     True
2024-06-02T22:32:58.2272487Z gitLab                          False
2024-06-02T22:32:58.2289433Z gitHubActions                   False
2024-06-02T22:32:58.2304263Z vsixFile                        
2024-06-02T22:32:58.2321470Z License file                    Specified
2024-06-02T22:32:58.2334601Z CodeSignCertPfxFile             Not specified
2024-06-02T22:32:58.2365798Z CodeSignCertPfxPassword         Not specified
2024-06-02T22:32:58.2408462Z CodeSignCertIsSelfSigned        False
2024-06-02T22:32:58.2419441Z KeyVaultCertPfxFile             Not specified
2024-06-02T22:32:58.2436815Z KeyVaultCertPfxPassword         Not specified
2024-06-02T22:32:58.2456608Z KeyVaultClientId                
2024-06-02T22:32:58.2469014Z BuildOutputFile                 
2024-06-02T22:32:58.2489580Z ContainerEventLogFile           
2024-06-02T22:32:58.2510497Z TestResultsFile                 D:\a\1\s\TestResults.xml
2024-06-02T22:32:58.2529848Z BcptTestResultsFile             D:\a\1\s\bcptTestResults.json
2024-06-02T22:32:58.2546718Z TestResultsFormat               JUnit
2024-06-02T22:32:58.2590596Z AdditionalCountries             
2024-06-02T22:32:58.2608322Z PackagesFolder                  D:\a\1\s\.packages
2024-06-02T22:32:58.2627853Z OutputFolder                    D:\a\1\s\.output
2024-06-02T22:32:58.2643736Z BuildArtifactFolder             
2024-06-02T22:32:58.2667604Z CreateRuntimePackages           False
2024-06-02T22:32:58.2683412Z AppVersion                      
2024-06-02T22:32:58.2702803Z AppBuild                        0
2024-06-02T22:32:58.2720880Z AppRevision                     0
2024-06-02T22:32:58.2735523Z SourceRepositoryUrl             
2024-06-02T22:32:58.2775533Z SourceCommit                    
2024-06-02T22:32:58.2795686Z BuildBy                         BcContainerHelper,6.0.19-preview1212
2024-06-02T22:32:58.2811887Z BuildUrl                        
2024-06-02T22:32:58.2825208Z Install Apps
2024-06-02T22:32:58.2841985Z - D:\a\1\s\artifacts\partnerx_partnerx Core_20.0.2024053159.1.app
2024-06-02T22:32:58.2849771Z Install Test Apps
2024-06-02T22:32:58.2859267Z - None
2024-06-02T22:32:58.2868369Z Previous Apps
2024-06-02T22:32:58.2877436Z - None
2024-06-02T22:32:58.2887736Z Application folders
2024-06-02T22:32:58.2902107Z - D:\a\1\s\HB projectx Base
2024-06-02T22:32:58.2909974Z Test application folders
2024-06-02T22:32:58.2920118Z - None
2024-06-02T22:32:58.2928585Z BCPT Test application folders
2024-06-02T22:32:58.2937756Z - None
2024-06-02T22:32:58.2946756Z BCPT Test suites
2024-06-02T22:32:58.2956683Z - None
2024-06-02T22:32:58.2965317Z Custom CodeCops
2024-06-02T22:32:58.2977472Z - None
2024-06-02T22:32:58.3102081Z 
2024-06-02T22:32:58.3102499Z   _____       _ _ _                                          _        _
2024-06-02T22:32:58.3104471Z  |  __ \     | | (_)                                        (_)      (_)
2024-06-02T22:32:58.3105135Z  | |__) |   _| | |_ _ __   __ _    __ _  ___ _ __   ___ _ __ _  ___   _ _ __ ___   __ _  __ _  ___
2024-06-02T22:32:58.3106276Z  |  ___/ | | | | | | '_ \ / _` |  / _` |/ _ \ '_ \ / _ \ '__| |/ __| | | '_ ` _ \ / _` |/ _` |/ _ \
2024-06-02T22:32:58.3106803Z  | |   | |_| | | | | | | | (_| | | (_| |  __/ | | |  __/ |  | | (__  | | | | | | | (_| | (_| |  __/
2024-06-02T22:32:58.3107298Z  |_|    \__,_|_|_|_|_| |_|\__, |  \__, |\___|_| |_|\___|_|  |_|\___| |_|_| |_| |_|\__,_|\__, |\___|
2024-06-02T22:32:58.3107699Z                            __/ |   __/ |                                                 __/ |
2024-06-02T22:32:58.3108107Z                           |___/   |___/                                                 |___/
2024-06-02T22:32:58.3108408Z 
2024-06-02T22:32:58.7825027Z Pulling mcr.microsoft.com/businesscentral:ltsc2022-dev
2024-06-02T22:37:33.8641274Z 
2024-06-02T22:37:33.8655481Z Pulling generic image took 276 seconds
2024-06-02T22:37:33.9353380Z 
2024-06-02T22:37:33.9354043Z    _____                _   _                               _        _
2024-06-02T22:37:33.9354259Z   / ____|              | | (_)                             | |      (_)
2024-06-02T22:37:33.9354487Z  | |     _ __ ___  __ _| |_ _ _ __   __ _    ___ ___  _ __ | |_ __ _ _ _ __   ___ _ __
2024-06-02T22:37:33.9356597Z  | |    | '__/ _ \/ _` | __| | '_ \ / _` |  / __/ _ \| '_ \| __/ _` | | '_ \ / _ \ '__|
2024-06-02T22:37:33.9361849Z  | |____| | |  __/ (_| | |_| | | | | (_| | | (__ (_) | | | | |_ (_| | | | | |  __/ |
2024-06-02T22:37:33.9362106Z   \_____|_|  \___|\__,_|\__|_|_| |_|\__, |  \___\___/|_| |_|\__\__,_|_|_| |_|\___|_|
2024-06-02T22:37:33.9362312Z                                      __/ |
2024-06-02T22:37:33.9362477Z                                     |___/
2024-06-02T22:37:33.9362537Z 
2024-06-02T22:37:34.5943171Z Creaing docker container
2024-06-02T22:37:35.3265636Z BcContainerHelper is version 6.0.19-preview1212
2024-06-02T22:37:35.3280509Z BcContainerHelper is running as administrator
2024-06-02T22:37:40.6803442Z HyperV is Enabled
2024-06-02T22:37:40.6830208Z Host is Microsoft Windows Server 2022 Datacenter - 10.0.20348.2461
2024-06-02T22:37:40.6883816Z UsePsSession is True
2024-06-02T22:37:40.6933925Z UsePwshForBc24 is True
2024-06-02T22:37:40.6947719Z UseWinRmSession is allow
2024-06-02T22:37:40.6993480Z UseSslForWinRmSession is True
2024-06-02T22:37:40.8562774Z Docker Client Version is 24.0.7
2024-06-02T22:37:40.8580123Z Docker Server Version is 24.0.7
2024-06-02T22:37:41.0772651Z Removing Desktop shortcuts
2024-06-02T22:37:41.3091028Z Downloading artifact /sandbox/25.0.20320.0/w1
2024-06-02T22:37:41.3837004Z Downloading C:\Users\VssAdministrator\AppData\Local\Temp\05dd25f1-b64d-4b0b-9d0c-330396ac257d.zip
2024-06-02T22:37:52.8750826Z Unpacking artifact to tmp folder using 7zip
2024-06-02T22:38:01.3542768Z Downloading platform artifact /sandbox/25.0.20320.0/platform
2024-06-02T22:38:01.3640952Z Downloading C:\Users\VssAdministrator\AppData\Local\Temp\33e44abd-337d-4373-a5a0-b3a365cafb1d.zip
2024-06-02T22:38:23.6937084Z Unpacking artifact to tmp folder using 7zip
2024-06-02T22:38:47.8973700Z Downloading Prerequisite Components
2024-06-02T22:38:47.9148734Z Downloading c:\bcartifacts.cache\sandbox\25.0.20320.0\platform\Prerequisite Components\IIS URL Rewrite Module\rewrite_2.0_rtw_x64.msi
2024-06-02T22:38:48.0501924Z Downloading c:\bcartifacts.cache\sandbox\25.0.20320.0\platform\Prerequisite Components\DotNetCore\DotNetCore.1.0.4_1.1.1-WindowsHosting.exe
2024-06-02T22:38:50.0848273Z Fetching all docker images
2024-06-02T22:38:50.6361775Z Fetching all docker volumes
2024-06-02T22:38:50.7837287Z Using image mcr.microsoft.com/businesscentral:ltsc2022-dev
2024-06-02T22:38:51.2539194Z Creating Container projectx-bld
2024-06-02T22:38:51.2550152Z Style: sandbox
2024-06-02T22:38:51.2560389Z Multitenant: Yes
2024-06-02T22:38:51.2571839Z Version: 25.0.20320.0
2024-06-02T22:38:51.2581799Z Platform: 25.0.20166.0
2024-06-02T22:38:51.2600333Z Generic Tag: 1.0.2.23-dev
2024-06-02T22:38:51.2639776Z Container OS Version: 10.0.20348.2461 (ltsc2022)
2024-06-02T22:38:51.2650651Z Host OS Version: 10.0.20348.2461 (ltsc2022)
2024-06-02T22:38:51.2758394Z Using process isolation
2024-06-02T22:38:51.2859630Z Using locale en-US
2024-06-02T22:38:51.2873805Z Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
2024-06-02T22:38:51.3016838Z Using license file D:\a\1\s\licenses\XXSBC24exp140824.bclicense
2024-06-02T22:38:51.3862064Z Additional Parameters:
2024-06-02T22:38:51.3880868Z --volume "D:\a\1\s:c:\sources"
2024-06-02T22:38:51.3889842Z --expose 5986
2024-06-02T22:38:51.3901588Z --env customNavSettings=EnableTaskScheduler=False
2024-06-02T22:38:51.3910767Z Files in C:\ProgramData\BcContainerHelper\Extensions\projectx-bld\my:
2024-06-02T22:38:51.3980167Z - AdditionalOutput.ps1
2024-06-02T22:38:51.3995448Z - AdditionalSetup.ps1
2024-06-02T22:38:51.4008127Z - HelperFunctions.ps1
2024-06-02T22:38:51.4023337Z - license.bclicense
2024-06-02T22:38:51.4032229Z - MainLoop.ps1
2024-06-02T22:38:51.4046898Z - SetupVariables.ps1
2024-06-02T22:38:51.4057048Z - updatehosts.ps1
2024-06-02T22:38:51.4094828Z Creating container projectx-bld from image mcr.microsoft.com/businesscentral:ltsc2022-dev
2024-06-02T22:38:52.8418436Z bfefd630b5ab42657195787692c5a505c863a0ee659fce1f51732934da58b6d6
2024-06-02T22:39:16.2725854Z Waiting for container projectx-bld to be ready
2024-06-02T22:39:52.3088273Z Using artifactUrl https://bcinsider-fvh2ekdjecfjd6gk.b02.azurefd.net/sandbox/25.0.20320.0/w1
2024-06-02T22:39:52.3093791Z Using installer from C:\Run\240
2024-06-02T22:39:52.3102505Z Installing Business Central: multitenant=True, installOnly=False, filesOnly=False, includeTestToolkit=False, includeTestLibrariesOnly=False, includeTestFrameworkOnly=False, includePerformanceToolkit=False, appArtifactPath=c:\dl\sandbox\25.0.20320.0\w1, platformArtifactPath=c:\dl\sandbox\25.0.20320.0\platform, databasePath=c:\dl\sandbox\25.0.20320.0\w1\BusinessCentral-W1.bak, licenseFilePath=c:\dl\sandbox\25.0.20320.0\w1\Cronus.bclicense, rebootContainer=True
2024-06-02T22:39:52.3117724Z Installing from artifacts
2024-06-02T22:40:04.1697982Z Starting Local SQL Server
2024-06-02T22:40:04.1704574Z WARNING: Waiting for service 'SQL Server (SQLEXPRESS) (MSSQL$SQLEXPRESS)' to 
2024-06-02T22:40:05.2375552Z start...
2024-06-02T22:40:06.3036211Z Starting Internet Information Server
2024-06-02T22:40:06.3037917Z Copying Service Tier Files
2024-06-02T22:40:11.2199985Z c:\dl\sandbox\25.0.20320.0\platform\ServiceTier\Program Files
2024-06-02T22:40:11.2205834Z c:\dl\sandbox\25.0.20320.0\platform\ServiceTier\System64Folder
2024-06-02T22:40:11.2206156Z Copying Web Client Files
2024-06-02T22:40:16.3482099Z c:\dl\sandbox\25.0.20320.0\platform\WebClient\Microsoft Dynamics NAV
2024-06-02T22:40:16.3485802Z Copying ModernDev Files
2024-06-02T22:40:16.3486143Z c:\dl\sandbox\25.0.20320.0\platform
2024-06-02T22:40:17.6233356Z c:\dl\sandbox\25.0.20320.0\platform\ModernDev\program files\Microsoft Dynamics NAV
2024-06-02T22:40:17.6234978Z Copying additional files
2024-06-02T22:40:17.6247655Z Copying ConfigurationPackages
2024-06-02T22:40:17.6248315Z C:\dl\sandbox\25.0.20320.0\platform\ConfigurationPackages
2024-06-02T22:40:17.6248729Z Copying Test Assemblies
2024-06-02T22:40:19.5864891Z C:\dl\sandbox\25.0.20320.0\platform\Test Assemblies
2024-06-02T22:40:19.5865434Z Copying Extensions
2024-06-02T22:40:19.5867984Z C:\dl\sandbox\25.0.20320.0\w1\Extensions
2024-06-02T22:40:19.5868198Z Copying Applications
2024-06-02T22:40:33.2722934Z C:\dl\sandbox\25.0.20320.0\platform\Applications
2024-06-02T22:40:33.2729558Z Copying dependencies
2024-06-02T22:40:34.6492955Z Importing PowerShell Modules
2024-06-02T22:40:47.5456323Z Restoring CRONUS Demo Database
2024-06-02T22:40:48.6508724Z Setting CompatibilityLevel for tenant on localhost\SQLEXPRESS
2024-06-02T22:41:27.8160242Z Exporting Application to CRONUS
2024-06-02T22:41:32.2263748Z Removing Application from tenant
2024-06-02T22:41:32.2264359Z Modifying Business Central Service Tier Config File for Docker
2024-06-02T22:41:55.5818397Z Creating Business Central Service Tier
2024-06-02T22:41:55.5995226Z Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
2024-06-02T22:41:55.5995815Z Starting Business Central Service Tier
2024-06-02T22:41:55.6056432Z Importing license file
2024-06-02T22:41:55.6056798Z Copying Database on localhost\SQLEXPRESS from tenant to default
2024-06-02T22:41:55.6057243Z Taking database tenant offline
2024-06-02T22:41:55.6057530Z Copying database files
2024-06-02T22:41:55.6057814Z Attaching files as new Database default
2024-06-02T22:41:55.6058125Z Putting database tenant back online
2024-06-02T22:41:55.6058349Z Mounting tenant database
2024-06-02T22:45:29.0294667Z Mounting Database for default on server localhost\SQLEXPRESS with AllowAppDatabaseWrite = False
2024-06-02T22:45:33.9242785Z Sync'ing Tenant
2024-06-02T22:45:33.9243369Z Tenant is Operational
2024-06-02T22:45:33.9243750Z Stopping Business Central Service Tier
2024-06-02T22:45:33.9243930Z Installation took 341 seconds
2024-06-02T22:45:33.9244148Z Installation complete
2024-06-02T22:45:35.7118014Z Initializing...
2024-06-02T22:45:35.7118417Z Setting host.containerhelper.internal to 172.19.112.1 in container hosts file
2024-06-02T22:45:35.7118798Z Starting Container
2024-06-02T22:45:35.7118994Z Hostname is projectx-bld
2024-06-02T22:45:36.7838251Z PublicDnsName is projectx-bld
2024-06-02T22:45:36.7840957Z Using NavUserPassword Authentication
2024-06-02T22:45:39.9752319Z Creating Self Signed Certificate
2024-06-02T22:45:39.9753583Z Self Signed Certificate Thumbprint 42C7E8E149386E829D1B9A1EDC64DE5F27F5E8C4
2024-06-02T22:45:39.9754432Z DNS identity projectx-bld
2024-06-02T22:45:41.0881463Z Modifying Service Tier Config File with Instance Specific Settings
2024-06-02T22:45:41.0882708Z Modifying Service Tier Config File with settings from environment variable
2024-06-02T22:45:43.2318555Z Setting EnableTaskScheduler to False
2024-06-02T22:45:49.7338189Z Starting Service Tier
2024-06-02T22:45:51.3775288Z Registering event sources
2024-06-02T22:45:51.3790945Z Creating DotNetCore Web Server Instance
2024-06-02T22:45:51.3812794Z Using application pool name: BC
2024-06-02T22:45:51.3826906Z Using default container name: NavWebApplicationContainer
2024-06-02T22:45:59.9646750Z Copy files to WWW root C:\inetpub\wwwroot\BC
2024-06-02T22:46:01.1560704Z Create the application pool BC
2024-06-02T22:46:04.6665066Z Create website: NavWebApplicationContainer without SSL
2024-06-02T22:46:04.6665563Z Update configuration: navsettings.json
2024-06-02T22:46:04.6688497Z Done Configuring Web Client
2024-06-02T22:46:04.6717514Z Enabling Financials User Experience
2024-06-02T22:46:04.6750456Z Using license file 'c:\run\my\license.bclicense'
2024-06-02T22:46:16.2428706Z Import License
2024-06-02T22:46:17.7070736Z Dismounting Tenant
2024-06-02T22:46:19.0107009Z Mounting Tenant
2024-06-02T22:46:23.6280819Z Mounting Database for default on server localhost\SQLEXPRESS with AllowAppDatabaseWrite = False
2024-06-02T22:46:28.1297928Z Sync'ing Tenant
2024-06-02T22:46:28.1303628Z Tenant is Operational
2024-06-02T22:46:29.5474205Z Creating http download site
2024-06-02T22:46:31.1281189Z Setting SA Password and enabling SA
2024-06-02T22:46:33.7890453Z Creating admin as SQL User and add to sysadmin
2024-06-02T22:46:35.2999385Z WARNING: This license is not compatible with this version of Business Central.
2024-06-02T22:46:38.6396931Z Creating SUPER user
2024-06-02T22:46:39.9588364Z WARNING: This license is not compatible with this version of Business Central.
2024-06-02T22:46:39.9589566Z WARNING: This license is not compatible with this version of Business Central.
2024-06-02T22:47:12.3583350Z Enable PSRemoting and setup user for winrm
2024-06-02T22:47:14.5268368Z Creating self-signed certificate for winrm
2024-06-02T22:47:14.5268914Z Container IP Address: 172.19.112.197
2024-06-02T22:47:14.5269094Z Container Hostname  : projectx-bld
2024-06-02T22:47:14.5269294Z Container Dns Name  : projectx-bld
2024-06-02T22:47:14.5270815Z Web Client          : http://projectx-bld/BC/?tenant=default
2024-06-02T22:47:14.5271106Z Dev. Server         : http://projectx-bld
2024-06-02T22:47:14.5271273Z Dev. ServerInstance : BC
2024-06-02T22:47:14.5275951Z Dev. Server Tenant  : default
2024-06-02T22:47:15.6729406Z Setting projectx-bld to 172.19.112.197 in host hosts file
2024-06-02T22:47:15.6730333Z Setting projectx-bld-default to 172.19.112.197 in host hosts file
2024-06-02T22:47:16.7357053Z Setting projectx-bld-default to 172.19.112.197 in container hosts file
2024-06-02T22:47:16.7357275Z 
2024-06-02T22:47:16.7357556Z Files:
2024-06-02T22:47:16.7768990Z http://projectx-bld:8080/ALLanguage.vsix
2024-06-02T22:47:16.7769511Z 
2024-06-02T22:47:16.7769936Z Container Total Physical Memory is 7.0Gb
2024-06-02T22:47:16.7770170Z Container Free Physical Memory is 2.3Gb
2024-06-02T22:47:16.7770250Z 
2024-06-02T22:47:16.7770438Z Initialization took 103 seconds
2024-06-02T22:47:16.7771447Z Ready for connections!
2024-06-02T22:47:21.8617303Z Reading CustomSettings.config from projectx-bld
2024-06-02T22:47:22.5968826Z Creating Desktop Shortcuts for projectx-bld
2024-06-02T22:47:23.2988701Z Cleanup old dotnet core assemblies
2024-06-02T22:47:23.3031232Z Container projectx-bld successfully created
2024-06-02T22:47:23.5837123Z 
2024-06-02T22:47:23.5939091Z Use:
2024-06-02T22:47:23.5963445Z Get-BcContainerEventLog -containerName projectx-bld to retrieve a snapshot of the event log from the container
2024-06-02T22:47:23.5985004Z Get-BcContainerDebugInfo -containerName projectx-bld to get debug information about the container
2024-06-02T22:47:23.6009136Z Enter-BcContainer -containerName projectx-bld to open a PowerShell prompt inside the container
2024-06-02T22:47:23.6032470Z Remove-BcContainer -containerName projectx-bld to remove the container again
2024-06-02T22:47:23.6057567Z docker logs projectx-bld to retrieve information about URL's again
2024-06-02T22:47:24.5749437Z 
2024-06-02T22:47:24.5750889Z Creating container took 591 seconds
2024-06-02T22:47:24.5781498Z 
2024-06-02T22:47:24.5787108Z  _____                _       _                   _                           _                 _
2024-06-02T22:47:24.5788009Z |  __ \              | |     (_)                 | |                         | |               (_)
2024-06-02T22:47:24.5793811Z | |__) |___ ___  ___ | |_   ___ _ __   __ _    __| | ___ _ __   ___ _ __   __| | ___ _ __   ___ _  ___ ___
2024-06-02T22:47:24.5795293Z |  _  // _ \ __|/ _ \| \ \ / / | '_ \ / _` |  / _` |/ _ \ '_ \ / _ \ '_ \ / _` |/ _ \ '_ \ / __| |/ _ \ __|
2024-06-02T22:47:24.5795594Z | | \ \  __\__ \ (_) | |\ V /| | | | | (_| | | (_| |  __/ |_) |  __/ | | | (_| |  __/ | | | (__| |  __\__ \
2024-06-02T22:47:24.5795876Z |_|  \_\___|___/\___/|_| \_/ |_|_| |_|\__, |  \__,_|\___| .__/ \___|_| |_|\__,_|\___|_| |_|\___|_|\___|___/
2024-06-02T22:47:24.5796058Z                                        __/ |            | |
2024-06-02T22:47:24.5796264Z                                       |___/             |_|
2024-06-02T22:47:24.5796326Z 
2024-06-02T22:47:24.7850511Z Sorted App folders
2024-06-02T22:47:24.7902138Z - D:\a\1\s\HB projectx Base
2024-06-02T22:47:24.8029632Z External dependencies
2024-06-02T22:47:24.8098077Z - 437dbf0e-84ff-417a-965d-ed2bb9650972:Microsoft_Base Application_23.0.0.0.app
2024-06-02T22:47:24.8152245Z - 63ca2fa4-4f03-4f2b-a480-172fef340d3f:Microsoft_System Application_23.0.0.0.app
2024-06-02T22:47:24.8162855Z Sorted TestApp folders
2024-06-02T22:47:24.8386095Z - None
2024-06-02T22:47:24.8670778Z 
2024-06-02T22:47:24.8688549Z   _____           _        _ _ _
2024-06-02T22:47:24.8690339Z  |_   _|         | |      | | (_)
2024-06-02T22:47:24.8696708Z    | |  _ __  ___| |_ __ _| | |_ _ __   __ _    __ _ _ __  _ __  ___
2024-06-02T22:47:24.8698309Z    | | | '_ \/ __| __/ _` | | | | '_ \ / _` |  / _` | '_ \| '_ \/ __|
2024-06-02T22:47:24.8699267Z   _| |_| | | \__ \ |_ (_| | | | | | | | (_| | | (_| | |_) | |_) \__ \
2024-06-02T22:47:24.8700271Z  |_____|_| |_|___/\__\__,_|_|_|_|_| |_|\__, |  \__,_| .__/| .__/|___/
2024-06-02T22:47:24.8701064Z                                         __/ |       | |   | |
2024-06-02T22:47:24.8701788Z                                        |___/        |_|   |_|
2024-06-02T22:47:24.8702347Z 
2024-06-02T22:47:25.7180538Z Run-AlPipeline Telemetry Correlation Id: c947f389-3dbe-4f6b-bfe5-6e05e89ff969
2024-06-02T22:47:25.9349544Z Measure-Command : Cannot bind argument to parameter 'Path' because it is an empty string.
2024-06-02T22:47:25.9355288Z At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\6.0.19\AppHandling\Run-AlPipeline.ps1:1180 char:1
2024-06-02T22:47:25.9363046Z + Measure-Command {
2024-06-02T22:47:25.9363320Z + ~~~~~~~~~~~~~~~~~
2024-06-02T22:47:25.9363739Z     + CategoryInfo          : InvalidData: (:) [Measure-Command], ParameterBindingValidationException
2024-06-02T22:47:25.9364048Z     + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Microsoft.PowerShell.Commands.Meas 
2024-06-02T22:47:25.9364323Z    ureCommandCommand
2024-06-02T22:47:25.9364495Z  
2024-06-02T22:47:28.2397395Z ##[error]PowerShell exited with code '1'.
2024-06-02T22:47:28.4019338Z ##[section]Finishing: PowerShell

```

--- a/AppHandling/Run-AlPipeline.ps1
+++ b/AppHandling/Run-AlPipeline.ps1
@@ -1183,7 +1183,9 @@ Measure-Command {
         Write-Host -ForegroundColor Yellow "Installing apps for additional country $testCountry"
     }
 
-    $dependenciesFolder = Join-Path $buildArtifactFolder "Dependencies"
+    if ($generateDependencyArtifact) {
+        $dependenciesFolder = Join-Path $buildArtifactFolder "Dependencies"
+    }
     $tmpAppFolder = Join-Path ([System.IO.Path]::GetTempPath()) ([guid]::NewGuid().ToString())
     $tmpAppFiles = @()
     $installApps | ForEach-Object{

